packages/trader/src/Constants/contract.tsx

```diff
@@ -0,0 +1,304 @@
+            position: 'bottom',
+        },
+        RUNHIGH: {
+            name: <Localize i18n_default_text='Only Ups' />,
+            position: 'top',
+        },
+        RUNLOW: {
+            name: <Localize i18n_default_text='Only Downs' />,
+            position: 'bottom',
+        },
+    } as const);
+
+// Config to display trade button and their position
+export const getSupportedContracts = (is_high_low: boolean) =>
+    ({
+        ACCU: {
+            button_name: <Localize i18n_default_text='Buy' />,
+            name: <Localize i18n_default_text='Accumulator' />,
+            position: 'top',
+        },
+        CALL: {
+            name: is_high_low ? <Localize i18n_default_text='Higher' /> : <Localize i18n_default_text='Rise' />,
+            position: 'top',
+        },
+        PUT: {
+            name: is_high_low ? <Localize i18n_default_text='Lower' /> : <Localize i18n_default_text='Fall' />,
+            position: 'bottom',
+        },
+        CALLE: {
+            name: <Localize i18n_default_text='Rise' />,
+            position: 'top',
+        },
+        PUTE: {
+            name: <Localize i18n_default_text='Fall' />,
+            position: 'bottom',
+        },
+        DIGITMATCH: {
+            name: <Localize i18n_default_text='Matches' />,
+            position: 'top',
+        },
+        DIGITDIFF: {
+            name: <Localize i18n_default_text='Differs' />,
+            position: 'bottom',
+        },
+        DIGITEVEN: {
+            name: <Localize i18n_default_text='Even' />,
+            position: 'top',
+        },
+        DIGITODD: {
+            name: <Localize i18n_default_text='Odd' />,
+            position: 'bottom',
+        },
+        DIGITOVER: {
+            name: <Localize i18n_default_text='Over' />,
+            position: 'top',
+        },
+        DIGITUNDER: {
+            name: <Localize i18n_default_text='Under' />,
+            position: 'bottom',
+        },
+        ONETOUCH: {
+            name: <Localize i18n_default_text='Touch' />,
+            position: 'top',
+        },
+        NOTOUCH: {
+            name: <Localize i18n_default_text='No Touch' />,
+            position: 'bottom',
+        },
+        MULTUP: {
+            name: <Localize i18n_default_text='Up' />,
+            position: 'top',
+        },
+        MULTDOWN: {
+            name: <Localize i18n_default_text='Down' />,
+            position: 'bottom',
+        },
+        VANILLALONGCALL: {
+            name: <Localize i18n_default_text='Call' />,
+            position: 'top',
+        },
+        VANILLALONGPUT: {
+            name: <Localize i18n_default_text='Put' />,
+            position: 'bottom',
+        },
+    } as const);
+
+export const getContractConfig = (is_high_low: boolean) => ({
+    ...getSupportedContracts(is_high_low),
+    ...getUnsupportedContracts(),
+});
+
+export type TContractsKeys = keyof ReturnType<typeof getContractConfig>;
```

Comment:
@kate-deriv let's move this type on top of the file )

Reply:
No problem)